src/Numerics/Mesh/Grids.jl

```diff
@@ -659,19 +659,24 @@ function computegeometry(
         )
     end
 
-    M = kron(1, ntuple(j -> ω[j], dim)...)
+    M = kron(1, reverse(ω)...)
```

Comment:
```suggestion
    # since `ξ1` is the fastest dimension and `ξdim` the slowest the tensor product order is reversed
   M = kron(1, reverse(ω)...)
```